### ISSUE TITLE
User can lose form data if they close tickets card whilst creating a new ticket

### ISSUE THREAD

**Issue by The-Daniel** [3drepo/3drepo.io#4664]
### Description

**This depends on #4563**

With recent ticket changes the user can close and reopen the tickets card and not lose the view they're on. This might make some users think that closing the card when creating a new ticket will retain all data but it doesn't.

There is an additional bug that if the user creates a pin and selects it, when they close and reopen the card the pin input will show selected but there is no pin set.

This was highlighted [here](https://github.com/3drepo/3drepo.io/pull/4653#discussion_r1425528329) 

- This is happening in: prod/**staging**/both 
- This requires a hotfix to production: yes/**no** 

### Steps to replicate
- Try to create a new ticket (fill in data)
- close the ticket form
- reopen the ticket form

### Current Behaviour
- The data is lost.

### Expected Behaviour
- <del> Perhaps a warning should be given that data will get lost if the card is closed </del>
- Or perhaps we can find a way to maintain the form data

### Pictures/ Gifs of the error
![image](https://github.com/3drepo/3drepo.io/assets/62481250/6fe0877e-f2e2-4a84-a22f-5d463fe4e411)
![image](https://github.com/3drepo/3drepo.io/assets/62481250/27b11f86-2f73-4def-9abd-dbb88db03461)
(notice the 'selected' pin)
